pkg/libvmi/vmi.go

```diff
@@ -289,6 +290,15 @@ func WithTPM(persistent bool) Option {
 	}
 }
 
+func WithUID(uid string) Option {
```

Comment:
```suggestion
func WithFirmwareUUID(uid string) Option {
```

Otherwise this might be confused with `spec.metadata.uid`. There are other PRs introducing an option with the same name for this field.

Reply:
fixed